src/winrtble/peripheral.rs

```diff
@@ -93,6 +94,7 @@ impl Peripheral {
                 adapter,
                 device: tokio::sync::Mutex::new(None),
                 address,
+                mtu: AtomicU16::new(23),
```

Comment:
Where does this 23 come from?

Reply:
It's the default size of the MTU prior to negotiation. It includes the L2CAP header (4 bytes), so usable size would be 19.

Reply:
I think I'll add an initial assignment from MaxPduSize prior to the event handler registration.

Reply:
Please add a constant for it, with a comment explaining this.